tests/Main.elm

```diff
@@ -0,0 +1,13 @@
+port module Main exposing (..)
```

Comment:
If it's an auto-generated file maybe we could add it to `.gitignore` and generate it when needed after the `git push`…